en/16_Multiple_Objects.adoc

```diff
@@ -0,0 +1,324 @@
+:pp: {plus}{plus}
+
+= Rendering Multiple Objects
+
+== Introduction
+
+In this chapter, we'll extend our Vulkan application to render multiple objects in the scene. So far, we've been rendering a single model, but real-world applications typically need to display many objects. This tutorial will show you how to efficiently manage and render multiple objects while reusing as many resources as possible.
+
+== Overview
+
+When rendering multiple objects, we need to consider which resources should be:
+1. *Shared across all objects* - to minimize memory usage and state changes
+2. *Duplicated for each object* - to allow for independent positioning and appearance
+
+Here's a quick reference for what typically falls into each category:
+
+*Shared resources:*
+* Vertex and index buffers (when objects use the same mesh)
+* Textures and samplers (when objects use the same textures)
+* Pipeline objects and pipeline layouts
+* Render passes
+* Command pools
+
+*Per-object resources:*
```

Comment:
This needs an additional line break between before the list, as the list is otherwise not rendered as such.